dev-python/yapf/yapf-0.29.0.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+inherit distutils-r1
+
+DESCRIPTION="A formatter for Python files"
+HOMEPAGE="https://github.com/google/yapf"
+SRC_URI="https://github.com/google/yapf/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}"
+
+python_test() {
+	esetup.py test
+}
```

Comment:
This can be dropped in favor of calling 'distutils_enable_tests setup.py' in global scope. If you are curious, please take a look at $repo/eclass/distutils-r1.eclass